Gopkg.toml

```diff
@@ -46,7 +46,7 @@ version="v1.4.7"
 
 [[constraint]]
   name = "knative.dev/serving"
-  version = "=v0.8.0"
+  version = "=v0.9"
```

Comment:
Hate to say this, but any reason we cant jump to 0.10?

Reply:
This is one is tested with test infrastructure et all pointed to Istio 1.3.1 and KNative 0.9. I will move this in, catch any edge cases in the meanwhile and follow-on quickly with 0.10

Reply:
I am about to send a PR to upgrade kubeflow knative manifests to 0.10, is it a quick change that just update to v0.10 here and e2e test to install 0.10?

Reply:
Here:
1. IBM KNative service is still on 0.8, and they are releasing 0.9 in Dec sometime. 
2. GKE is also running on 0.9 currently
2. Our internal environments are either 0.9, or 0.8, and at this point, I would need to verify the feasibility of 1.0 before taking a final call
3. Also took 7-8 days to coordinate, get test infra and environments updated, test both manually and in automated fashion. 

I will roll this in, and in next one I will jump to 0.10 immediately in form of a PR, have @jinchihe upgrade the test infra in another PR, and we also update the environments on our side to run tests and also verify manually. 

In parallel, I would get the feasibility date from IBM and GCP side, before taking a final call to jump to 0.10 immediately